src/services/companyLookup.service.js

```diff
@@ -28,6 +36,18 @@ module.exports = class CompanyLookupService {
         }
       })
 
-    return companyName
+    return company
+  }
+
+  static async getCompanyName (companyNumber) {
```

Comment:
I think we should have a single getCompany() method that returns a single object containing the three required properties of companyName, companyStatus and isActive flags. The reason being that otherwise we will hit the companies house service for each item of company data that we want to display on the page.

So for example, wherever we are currently saying:

getCompanyName(...) we should be able to say getCompany(...).name